jenkins-schema.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash -xe
+export DISPLAY=:99
+export GOVUK_APP_DOMAIN=test.gov.uk
+export GOVUK_ASSET_ROOT=http://static.test.gov.uk
+export GITHUB_STATUS_CONTEXT="verify government-frontend against content format examples"
+env
+
+function github_status {
+  STATUS="$1"
+  MESSAGE="$2"
+  if [ "$GIT_BRANCH" != "origin/master" ]; then
+    GHTOOLS_DEBUG=True gh-status alphagov/govuk-content-schemas "$GIT_COMMIT" "$STATUS" -d "Build #${BUILD_NUMBER} ${MESSAGE}" -u "$BUILD_URL" -c "$GITHUB_STATUS_CONTEXT" >/dev/null
```

Comment:
Sorry, that `GHTOOLS_DEBUG=True` prefix shouldn't be there - I'll do a PR to remove it from the Whitehall version.